packages/dashboard/src/PanelManager.ts

```diff
@@ -306,11 +314,9 @@ class PanelManager {
 
   addClosedPanel(glContainer: GoldenLayout.Container): void {
     const config = LayoutUtils.getComponentConfigFromContainer(glContainer);
-    if (config) {
-      const dehydratedConfig = this.dehydrateComponent(
-        config.component,
-        config
-      );
+    let dehydratedConfig;
+    if (config && isReactComponentConfig(config)) {
+      dehydratedConfig = this.dehydrateComponent(config.component, config);
```

Comment:
Not sure why you moved `dehydratedConfig` outside of the if:
```suggestion
    if (config && isReactComponentConfig(config)) {
      const dehydratedConfig = this.dehydrateComponent(config.component, config);
```